devops/db/liquibase/changelogs/payment-transaction-gateway/incremental-changelogs/V5_PM_988_changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.8.xsd">
+    <changeSet id="V7__PM_988_ADD_COLUMN_STATUS" author="liquibase" failOnError="true" context="incremental">
```

Comment:
@andrearovere the changeset id should be the same as the incremental-changelog name
I pushed some fixes